src/Tools/dotnet-monitor/HostBuilder/HostBuilderHelper.cs

```diff
@@ -109,96 +109,6 @@ public static IHostBuilder CreateHostBuilder(HostBuilderSettings settings)
 
                         builder.AddCommandLine(arguments.ToArray());
                     }
-                })
-                //Note this is necessary for config only because Kestrel configuration
-                //is not added until WebHostDefaults are added.
-                .ConfigureWebHostDefaults(webBuilder =>
```

Comment:
This mostly moved to the CollectCommandHandler and the dynamic configuration source for blocking the `urls` key has been completely removed.